### ISSUE TITLE
Can You Implement this Controller Plugin?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [facelift/mupen64gc#117]
```
Hello Emulate Mii team. thanks for your awesome work. I just wanted to make
a sugestion, maybe you can contact N-Rage to use the N-Rage controller
Plugin, which is the best customizable I've seen in a N64 emulator, and
also has transfer pack support by using a GB or GBC rom and a .sav battery
file from that rom, that'll be a great addition to this great wii emulator.
Thanks for your work I hope to donate soon to your project. Thanks again
```

Original issue reported on code.google.com by `megaman...@gmail.com` on 2 Oct 2009 at 2:43
- Merged into: #128

**Comment by GoogleCodeExporter** [comment]
```
Sorry to put this request like an Issue but I'm new at using code google, my 
most
sinceriest apologies. Keep up the awesome work!
```

Original comment by `megaman...@gmail.com` on 2 Oct 2009 at 2:46

**Comment by GoogleCodeExporter** [comment]
```
We will certainly have configurable button mapping in the future, but without
hardware available, it's quite hard to do N64 pak support such as mempak's and
transfter paks. 

Even the current N64 to GC adapters most likely don't allow this.
```

Original comment by `emuki...@gmail.com` on 2 Oct 2009 at 2:52
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
This issue should be merged into issue #105 or issue #128.
```

Original comment by `Gamewiza...@gmail.com` on 4 Oct 2009 at 4:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `emuki...@gmail.com` on 5 Oct 2009 at 2:16
- Changed state: **Duplicate**
